### ISSUE TITLE
ArgoCD does not show all helm parameters from the values.yaml file

### ISSUE THREAD

**Issue by LiberoScarcelli** [argoproj/argo-cd#10020]
Hi,

it seems that ArgoCD is not showing all parameters from values.yaml files. This is what the text editor shows:

![helm](https://user-images.githubusercontent.com/101608253/179479691-bce9af29-84c3-4641-b951-0ea14b06c5f1.jpg)

This is what the ArgoCD console shows:

![argocd](https://user-images.githubusercontent.com/101608253/179479800-9cddb3eb-641a-41cd-a7f1-2064a360777e.jpg)

The "scripts" parameter is missing

**Comment by crenshaw-dev** [comment]
I think this might be a duplicate of https://github.com/argoproj/argo-cd/issues/9213

**Comment by crenshaw-dev** [comment]
This might be because it expects an object type rather than a string, number, or bool. We'd need to add a YAML editor to express those fields.

**Comment by LiberoScarcelli** [comment]
Yes exactly, as this parameter expects a multi-lines value, an entire script to initialize the DB. Thank you!
